### PR TITLE
WIP: bazel build for tt-mlir

### PR DESCRIPTION
**Pull request by AleksKnezevic**

### CODE REVIEW

File: include/ttmlir/Version.h [hunk 1]
```diff
@@ -8,16 +8,20 @@
...
 #endif
 #ifndef TTMLIR_VERSION_MINOR
-#error "TTMLIR_VERSION_MINOR must be defined"
+#define TTMLIR_VERSION_MINOR 0
+// #error "TTMLIR_VERSION_MINOR must be defined"
 #endif
 #ifndef TTMLIR_VERSION_PATCH
-#error "TTMLIR_VERSION_PATCH must be defined"
+#define TTMLIR_VERSION_PATCH 0
+// #error "TTMLIR_VERSION_PATCH must be defined"
```
[the remark this answers is not in the document]
Comment (AleksKnezevic) [reply include/ttmlir/Version.h:24]: I'll get it back. This PR is not ready to merge, just opened it to run CI,

**Comment by nsmithtt** [comment]
Ubuntu build is failing with:
```
E: Unable to locate package bazel-6.5.0
```

Strangely the Bazel build is running, so some kind of Bazel must be preinstalled.

**Comment by AleksKnezevic** [comment]
Yeah, seems to download it as part of the build:

`2024/07/25 16:36:49 Downloading https://releases.bazel.build/7.2.1/release/bazel-7.2.1-linux-x86_64...`

6.5 is needed for xla

**Comment by nsmithtt** [comment]
Closing, we ended up building a standalone pjrt device.

**Pull request closed without merging**